src/blog/jwt-identification-and-hot-streams/index.md

```diff
@@ -0,0 +1,172 @@
+# AnyCable PRO:<br>JWT identification and "hot streams"
+
+September 27, 2021
+{date}
+
+When we started collecting applications for the [AnyCable PRO][pro] early access program back in July, we asked respondents to share the way they use _cables_ and which features they would like us to provide out-of-the-box. The analysis of their responses led to the development of two new PRO features, [JWT identification][jwt-id-docs] and [Signed streams support][signed-streams-docs], which might seem unrelated to each other from the first look but turned out to bring a huge performance boost when used together. Continue reading to learn more!
+{intro}
+
+<div class="divider"></div>
+
+## Dealing with authentication
+
+Among the requested features, one of the leaders was "a token-based authentication". Most Action Cable applications relies on cookies as an authentication mechanism (directly or via sessions). The main benefit of the cookie-based authentication is the simplicity: it just works. However, there are some drawbacks:
+
+- Authenticating non-web clients (e.g., mobile apps) is becoming cumbersome.
```

Comment:
"is becoming" or "has become"? The thing is, "is becoming" sounds odd.

Reply:
"Becomes"?

Reply:
No, it cannot be "becomes", that would be like the feeling in Japanese poem, if that makes any sense.

The primary question is -- is this still happening, or has it passed the point of no return -- is it cumbersome already, or is it in the process of becoming cumbersome?

Reply:
What I wanted to say here is that authenticating mobile clients via cookies is complicated.

So, yeah, it's probably just "is", there is no process or whatever.

Reply:
yep, just delete "becoming" then

Reply:
```suggestion
- Authenticating non-web clients (e.g., mobile apps) is cumbersome.
```